cmd/signature-v2.go

```diff
@@ -85,8 +85,12 @@ func doesPresignV2SignatureMatch(r *http.Request) APIErrorCode {
 	cred := serverConfig.GetCredential()
 
 	// r.RequestURI will have raw encoded URI as sent by the client.
-	splits := splitStr(r.RequestURI, "?", 2)
-	encodedResource, encodedQuery := splits[0], splits[1]
+	tokens := strings.SplitN(r.RequestURI, "?", 2)
```

Comment:
> I removed splitStr() as its used in two places only and used exactly for two element tokens. splitStr() is necessary utility function.

I guess the real reason is that you don't understand the exact purpose of splitStr function from its name and the code is actually more readable without it, right?